fms_fsdp/config/training.py

```diff
@@ -32,13 +33,17 @@ class train_config:
     grad_clip_thresh: float = 1.0
 
     # profiling and logging
+    report_interval: int = 200
+    checkpoint_interval: int = 20000
     use_profiler: bool = False
     use_wandb: bool = False
     wandb_dir: str = "/lustre/lchu/fms-fsdp"
-    wandb_project_name = f"llama-{model_variant}"
-    wandb_run_id: str = "aabbccdd"  # give a unique id per job, for resume purpose
-    report_interval: int = 200
-    checkpoint_interval: int = 20000
+    wandb_project_name: str = "llama"  # project name for a group of runs
+    wandb_run_id: str = "aabbccdd"  # give a unique id per run, for job resume purpose
+    use_aim: bool = False
```

Comment:
can we make this more generic? e.g. we could have `tracker: str = "wandb"` (or aimstack etc), ... `tracker_project_name` (instead of both aim_project_name, wandb_project_name, etc).

Reply:
@nairbv good point.